rootfs/scheduler/__init__.py

```diff
@@ -605,11 +605,11 @@ def _create_rc(self, name, image, command, **kwargs):  # noqa
         TEMPLATE = RCD_TEMPLATE
 
         # Check if it is a slug builder image.
-        # Example format: golden-earrings:git-5450cbcdaaf9afe6fadd219c94ac9c449bd62413s
         if kwargs.get('build_type', {}) == "buildpack":
```

Comment:
If we are checking for slug_url we don't require a check for buildtype here . Just a check to see if slug_url is present or not is sufficient